src/main/java/com/example/hop_oasis/controller/CiderController.java

```diff
@@ -34,45 +32,25 @@ public ResponseEntity<Page<CiderInfoDto>> getAllCiders(@RequestParam(value = "pa
                                                            @RequestParam(value = "size",defaultValue = "10") int size) {
         Page<CiderInfoDto> ciderPage = ciderService.getAllCiders(PageRequest.of(page, size));
         HttpHeaders headers = new HttpHeaders();
-        headers.add("Content-Range", "items " + page * size + "-" + ((page + 1) * size - 1) + "/" + ciderPage.getTotalElements());
+        headers.add("Content-Range",
```

Comment:
since you return Page, Content-Range header is redundant. You can simply delete this line.